richeditor-compose/src/commonMain/kotlin/com/mohamedrejeb/richeditor/model/RichTextState.kt

```diff
@@ -615,8 +649,13 @@ class RichTextState internal constructor(
         )
     }
 
+    @Deprecated("Use toggleOrderedList() instead to add or remove ordered list")
     fun removeOrderedList() {
         val paragraph = getRichParagraphByTextIndex(selection.min - 1) ?: return
+        removeOrderedList(paragraph)
```

Comment:
Same as above, this shouldn't be deprecated.

Reply:
ready